src/main/java/frc/robot/RobotMap.java

```diff
@@ -28,26 +29,22 @@
   // number and the module. For example you with a rangefinder:
   // public static int rangefinderPort = 1;
   // public static int rangefinderModule = 1;
-  public static final Spark leftFront = new Spark(0);// TODO: Set to actual port numbers
```

Comment:
In CommandBase, the objects are created within their subsystems, not in RobotMap.   RobotMap should only be defining static final integers specifying ports

Reply:
How are you going to make sure there's only ever one object associated with a port then? Also, in the [FRC Docs](https://wpilib.screenstepslive.com/s/currentCS/m/java/l/599735-simple-subsystems) they use `Victor motor = RobotMap.clawMotor;`

Reply:
Ok, just a different way to do it I guess.   I've always seen this in subsystem:      Victor motor = new Victor(RobotMap.clawMotor);